apps/connect/src/components/modal/modals/settings/package-manager.tsx

```diff
@@ -119,10 +125,23 @@ export const PackageManager: React.FC = () => {
 
     return (
         <BasePackageManager
-            mutable={isMutable}
+            mutable={true}
```

Comment:
we need to clarify what is up with this `mutable` property. all it does is check if hmr is available.